_topic_maps/_topic_map_rosa_hcp.yml

```diff
@@ -185,713 +118,31 @@ Topics:
-#   #  Distros: openshift-rosa-hcp
-#   - Name: OpenShift CLI developer command reference
-#     File: developer-cli-commands
-#   - Name: OpenShift CLI administrator command reference
-#     File: administrator-cli-commands
-#     Distros: openshift-rosa-hcp
-# - Name: Developer CLI (odo)
-#   File: odo-important-update
-#   # Dir: developer_cli_odo
-#   Distros: openshift-rosa-hcp
-#   # Topics:
-#   # - Name: odo release notes
-#   #  File: odo-release-notes
-#   # - Name: Understanding odo
-#   #  File: understanding-odo
-#   # - Name: Installing odo
-#   #  File: installing-odo
-#   # - Name: Configuring the odo CLI
-#   #  File: configuring-the-odo-cli
-#   # - Name: odo CLI reference
-#   #  File: odo-cli-reference
-# - Name: Knative CLI (kn) for use with OpenShift Serverless
-#   File: kn-cli-tools
-#   Distros: openshift-rosa-hcp
-# - Name: Pipelines CLI (tkn)
-#   Dir: tkn_cli
-#   Distros: openshift-rosa-hcp
-#   Topics:
-#   - Name: Installing tkn
-#     File: installing-tkn
-#   - Name: Configuring tkn
-#     File: op-configuring-tkn
-#   - Name: Basic tkn commands
-#     File: op-tkn-reference
-# - Name: opm CLI
-#   Dir: opm
-#   Distros: openshift-rosa-hcp
-#   Topics:
-#   - Name: Installing the opm CLI
-#     File: cli-opm-install
-#   - Name: opm CLI reference
-#     File: cli-opm-ref
-# - Name: Operator SDK
-#   Dir: osdk
-#   Distros: openshift-rosa-hcp
-#   Topics:
-#   - Name: Installing the Operator SDK CLI
-#     File: cli-osdk-install
-#   - Name: Operator SDK CLI reference
-#     File: cli-osdk-ref
-# - Name: ROSA CLI
-#   Dir: rosa_cli
-#   Distros: openshift-rosa-hcp
-#   Topics:
-#   # - Name: CLI and web console
-#   #   File: rosa-cli-openshift-console
-#   - Name: Getting started with the ROSA CLI
-#     File: rosa-get-started-cli
-#   - Name: Managing objects with the ROSA CLI
-#     File: rosa-manage-objects-cli
-#   - Name: Checking account and version information with the ROSA CLI
-#     File: rosa-checking-acct-version-cli
-#   - Name: Checking logs with the ROSA CLI
-#     File: rosa-checking-logs-cli
-#   - Name: Least privilege permissions for ROSA CLI commands
-#     File: rosa-cli-permission-examples
-#
-# ---
-# Name: Red Hat OpenShift Cluster Manager
-# Dir: ocm
-# Distros: openshift-rosa-hcp
-# Topics:
-# - Name: Red Hat OpenShift Cluster Manager
-#   File: ocm-overview
-#  - Name: Red Hat OpenShift Cluster Manager
-#    File: ocm-overview
-#  - Name: Using the OpenShift web console
-#    File: rosa-using-openshift-console
 ---
 Name: Cluster administration
 Dir: rosa_cluster_admin
 Distros: openshift-rosa-hcp
 Topics:
-#  - Name: Cluster configurations
-#    File: rosa-cluster-config
-#  - Name: Cluster authentication
-#    File: rosa-cluster-auth
-#  - Name: Authorization and RBAC
-#    File: rosa-auth-rbac
-- Name: Cluster notifications
-  File: rosa-cluster-notifications
-  Distros: openshift-rosa-hcp
-- Name: Cluster configurations
-  File: rosa-cluster-notifications
----
```

Comment:
From here to line 517 is duplicated content - the files down on line 531 and 566 are supposed to be in the cluster admin doc, but they got inclded in this duplicate security book instead because the next book (lines 521-524, also duplicated from 497-501??) was commented out